src/sentry/tagstore/snuba/backend.py

```diff
@@ -420,7 +391,8 @@ def get_group_tag_keys_and_top_values(
             conditions.append(['tags_key', 'NOT IN', self.EXCLUDE_TAG_KEYS])
 
         values_by_key = snuba.query(
-            start, end, ['tags_key', 'tags_value'], conditions, filters, aggregations,
+            kwargs.get('start'), kwargs.get('end'), [
+                'tags_key', 'tags_value'], conditions, filters, aggregations,
```

Comment:
Using `kwargs` like this is kind of an anti-pattern but since you didn't introduce it, it is probably better to remain consistent with the existing code and not increase the scope of the change, so this is okay. Mostly pointing this out to say please don't take this as an example of good coding practice

Reply:
Good to know! When I do want to extract arguments from `kwargs`, is it better to extract it in the params?
```
get_group_tag_keys_and_top_values(...start=None, end=None)
```

Reply:
Yes — for example in this case, if `end=xyz` was accidentally replaced with `stop=xyz` or some other invalid argument, the method would accept the invalid keyword argument, but not use it. This makes it difficult to identify incorrect usage, especially when refactoring